### PR TITLE
Add diet and cuisine blocks

### PR DESCRIPTION
**Pull request by GuillaumeGomez**

### CODE REVIEW

File: idunn/blocks/services_and_information.py [hunk 1]
```diff
@@ -94,11 +94,77 @@ def from_es(cls, es_poi, lang):
...
+    STATUS_ONLY = "only"
+
+    value = validators.String(
+        enum=[STATUS_YES, STATUS_NO, STATUS_ONLY]
+    )
+
+    @classmethod
+    def from_es(cls, info, lang):
+        info = info.split('=')
+        if info[0] not in ("pescetarian", "vegetarian", "lacto_vegetarian", "ovo_vegetarian",
```
Comment (GuillaumeGomez) [inline idunn/blocks/services_and_information.py]: To be reduced to "vegetarian", "vegan" and "gluten_free".

File: idunn/blocks/services_and_information.py [hunk 2]
```diff
@@ -94,11 +94,77 @@ def from_es(cls, es_poi, lang):
...
+
+
+class DietBlock(BaseBlock):
+    BLOCK_TYPE = "diet"
+
+    diets = validators.Array(items=Diet)
+
+    @classmethod
+    def from_es(cls, es_poi, lang):
+        diet = es_poi.get("properties", {}).get("diet")
```
Comment (GuillaumeGomez) [inline idunn/blocks/services_and_information.py]: The key should be "diet:[diet_kind]".

File: idunn/blocks/services_and_information.py [hunk 3]
```diff
@@ -94,11 +94,64 @@ def from_es(cls, es_poi, lang):
...
+
+
+class CuisineBlock(BaseBlock):
+    BLOCK_TYPE = "cuisine"
+    SUPPORTED_DIETS = ("vegetarian", "vegan", "gluten_free")
+
+    cuisines = validators.Array(items=Cuisine)
+    vegetarian_status = validators.Object()
+    vegan_status = validators.Object()
+    gluten_free_status = validators.Object()
```
Comment (amatissart) [inline idunn/blocks/services_and_information.py]: Any reason to use an object here ? 
Wouldn't `"vegetarian": STATUS_YES` be more simple, with a string value among all statuses ?

File: idunn/blocks/services_and_information.py [hunk 4]
```diff
@@ -94,11 +94,64 @@ def from_es(cls, es_poi, lang):
...
+
+    @classmethod
+    def from_es(cls, es_poi, lang):
+        cuisine = es_poi.get("properties", {}).get("cuisine")
+
+        if cuisine is None:
+            return None
```
Comment (amatissart) [inline idunn/blocks/services_and_information.py]: I think we may want to return other details ("vegetarian", etc) that could be present, even if "cuisine" is absent.

File: tests/cuisine.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
+from app import app
+from idunn.blocks.services_and_information import CuisineBlock, Cuisine
+
+def test_internet_access_block():
```
Comment (amatissart) [inline tests/cuisine.py]: testcase to rename

**Comment by GuillaumeGomez** [comment]
Updated.

**Pull request merged**